osquery/tables/utility/file.cpp

```diff
@@ -99,6 +127,171 @@ void genFileInfo(const fs::path& path,
     r["type"] = "unknown";
   }
 
+#else
+
+  // Get the handle of the file object.
+  hFile = CreateFile(TEXT(path.string().c_str()),
+                     GENERIC_READ,
+                     FILE_SHARE_READ,
+                     NULL,
+                     OPEN_EXISTING,
+                     FILE_ATTRIBUTE_NORMAL,
+                     NULL);
+
+  // Check GetLastError for CreateFile error code.
+  if (hFile == INVALID_HANDLE_VALUE) {
+    return;
+  }
+
+  // Get the owner SID of the file.
+  dwRtnCode = GetSecurityInfo(hFile,
+                              SE_FILE_OBJECT,
+                              OWNER_SECURITY_INFORMATION,
+                              &pSidOwner,
+                              NULL,
+                              NULL,
+                              NULL,
+                              &pSD);
+
+  // Check GetLastError for GetSecurityInfo error condition.
+  if (dwRtnCode != ERROR_SUCCESS) {
+    return;
+  }
+
+  FILE_BASIC_INFO basicInfo;
+  BY_HANDLE_FILE_INFORMATION fileInfo;
+
+  if (0 == GetFileInformationByHandle(hFile, &fileInfo)) {
+    return;
+  }
+
+  auto fileIndex = ((unsigned long long)fileInfo.nFileIndexHigh << 32) +
```

Comment:
Avoid implicit C-style casting. Here you should be able to do `static_cast<unsigned long long>()`

Reply:
nit: similarly here and throughout, use `snake_case` for local variables